internal/cli/alerts/unacknowledge.go

```diff
@@ -39,14 +39,18 @@ func (opts *UnacknowledgeOpts) initStore() error {
 	return err
 }
 
+var unackTemplate = `ID	TYPE_NAME	METRIC_NAME
+{{.ID}}	{{.EventTypeName}}	{{.MetricName}}
```

Comment:
this is an "action" kind of operation, similar to what we were doing with delete, and now with create and update we can make this be something like
```suggestion
var unackTemplate = "Alert {{.ID}} un acknowledged"
```

or something like that tho I'd check with the ui what do we say as `un acknowledged` is not an actual word